Source/DafnyCore/Compilers/SinglePassCompiler.cs

```diff
@@ -64,6 +66,23 @@ public abstract class SinglePassCompiler : Plugins.Compiler {
 
     public CoverageInstrumenter Coverage;
 
+    public virtual int RunProcess(ProcessStartInfo psi, Process process, String platform, TextWriter outputWriter) {
+      try {
+        if (process == null) {
+          process = Process.Start(psi);
+          if (process == null) {
+            return -1;
+          }
+        }
+        process.WaitForExit();
+        return process.ExitCode;
+      } catch (System.ComponentModel.Win32Exception e) {
```

Comment:
This looks suspicious: Win32?

Reply:
It is what was in the code before -- do you have a suggestion?

Reply:
And it is what current MS documentation says for Start() and WaitForExit()

Reply:
Thanks!  How odd.